### ISSUE TITLE
daily build tarball pointing to the wrong branch?

### ISSUE THREAD

**Issue by greyltc** [nextcloud/nextcloud.com#115]
Hi, on your releases page, https://nextcloud.com/install/ you have a link to your daily build tarball: https://download.nextcloud.com/server/daily/latest.tar.bz2

It seems that this is somehow out of sync with the current development. The daily build tarball builds 9.0.52. it would probably be more relevant to have the daily build tarball be on the 10.0 branch, right?

Could you please keep the daily build release tarball pointing to the branch you'd like us to be testing?
Thanks!
https://github.com/greyltc/docker-nextcloud/issues/5

**Comment by jospoortvliet** [comment]
Yeah, so the script right now points to the latest in stable 9, should move to stable 10. But I'm not super happy with it either, so let's see if we can create an issue and then convince @LukasReschke to spend a little time on this.

What I'd like to have is a latest-stable daily and a master daily. Makes sense, right?
